### PR TITLE
Fix for jQueryUI context menu 17.1

### PR DESCRIPTION
**Pull request by desig9stein**

### CODE REVIEW

File: src/css/themes/bootstrap3/yeti/infragistics.theme.css [hunk 1]
```diff
@@ -1149,7 +1149,8 @@ body .ui-tooltip {
 .ui-icon-carat-1-ne {
   background-position: -16px 0;
 }
-.ui-icon-carat-1-e {
+.ui-icon-carat-1-e,
```
Comment (kdinev) [inline src/css/themes/bootstrap3/yeti/infragistics.theme.css]: Why is this necessary? Is it a workaround for something?

Comment (desig9stein) [reply src/css/themes/bootstrap3/yeti/infragistics.theme.css]: the class “ui-icon-caret-1-e” in the new version of jQueryUI it’s different from the one “ui-icon-carat-1-e” in the old version. 
Now I know it seems odd but there is actually a different ASCI symbol in the class from the new version

Comment (kdinev) [reply src/css/themes/bootstrap3/yeti/infragistics.theme.css]: OMG. I didn't see the `carAt` vs. `carEt`.

**Comment by Lipata** [comment]
Taking the latest SR for 17.1 (2029) and using the ccs folder with the changes from this pull requests adds the arrows for the sub-menu, but there is another visual problem:

image

File: src/css/themes/bootstrap3/yeti/infragistics.theme.css (hunk 1 above)
Comment (desig9stein) [reply src/css/themes/bootstrap3/yeti/infragistics.theme.css]: lol, you are not the only one :)
